ZACwire.h

```diff
@@ -114,15 +149,16 @@ class ZACwire {
 				static unsigned int deltaTime;
 				deltaTime = microtime - deltaTime;			//measure time to previous rising edge
 				if (deltaTime >> 10) {					//start bit
-					backUP ^= bitCounter == 20;
+					backUP ^= bitCounter >= 20;     //sometimes additional interrupts are thrown afterwards

```

Comment:
You are right: This is a left over from previous tests. But still the code is not ideal with "==" either because then "backUP temp" which got collected far in the past is still returned in getTemp().
The new _directMode will also fix this, but perhaps it would be better to add some more logic to stop reading bits after the 19th bit is read? I have seen random, non-deterministic interrupts every now and then (quite often!) on esp32.